src/mmrelay/matrix/prefixes.py

```
@@ -0,0 +1,600 @@
+    prefix_format = (
+        str(prefix_format_value)
+        if prefix_format_value is not None
+        else DEFAULT_MESHTASTIC_PREFIX
+    )
+
+    # Parse username and server from user_id if available
+    username = ""
+    server = ""
+    if user_id:
+        # Extract username and server from @username:server.com format
+        if user_id.startswith("@") and ":" in user_id:
```

Ruff (0.15.9)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)